WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Extensions.swift

```diff
@@ -288,8 +288,12 @@ extension NotificationsViewController: WPTableViewHandlerDelegate
         // Update Separators:
         // Due to an UIKit bug, we need to draw our own separators (Issue #2845). Let's update the separator status
         // after a DB OP. This loop has been measured in the order of milliseconds (iPad Mini)
+
         for indexPath in tableView.indexPathsForVisibleRows ?? [] {
-            let cell = tableView.cellForRowAtIndexPath(indexPath) as! NoteTableViewCell
+            guard let cell = tableView.cellForRowAtIndexPath(indexPath) as? NoteTableViewCell else {
+                continue
```

Comment:
Thanks for chiming in Diego!

Real problem here is:

```
public func cellForRowAtIndexPath(indexPath: NSIndexPath) -> UITableViewCell? 
// returns nil if cell is not visible or index path is out of range
```

`cellForRowAtIndexPath` may return nil. Although cells are always expected to be `NoteTableViewCell` instances... i didn't consider that we might be dealing with a nil, to begin with.